src/main/kotlin/com/terraformation/backend/documentproducer/db/VariableStore.kt

```diff
@@ -132,16 +132,11 @@ class VariableStore(
       with(VARIABLES) {
         dslContext
             .select(ID)
+            .on(STABLE_ID)
```

Comment:
Huh, didn't know about this. I've been using `.distinctOn()`.

Reply:
I may change it over to that. I was having issues with that at first, but just hadn't understood the sorting needed. The `.on` is still using a `distinct on` under the hood, so might be better to be more explicit for readability and searchability.